aws-lc-rs/src/fips.rs

```diff
@@ -126,7 +126,11 @@ pub(crate) use indicator_check;
 macro_rules! check_fips_service_status {
     ($function:expr) => {{
         use $crate::fips::get_fips_service_status;
+        // Clear the current indicator status first by retrieving it
```

Comment:
Should we clear an unchecked indicator?  If this next call succeeds, they might never know there was a previous "unapproved" operations.

Reply:
The net result is the same right? If the indicator was already set to a value and you make a call, you don't know if it was the previous value or the current one. Unless you had explicitly checked it.